yarn-project/p2p/src/services/reqresp/rate-limiter/rate_limits.ts

```diff
@@ -22,6 +22,16 @@ export const DEFAULT_RATE_LIMITS: ReqRespSubProtocolRateLimits = {
       quotaCount: 10,
     },
   },
+  [ReqRespSubProtocol.AUTH]: {
+    peerLimit: {
+      quotaTimeMs: 1000,
+      quotaCount: 5,
+    },
+    globalLimit: {
+      quotaTimeMs: 1000,
+      quotaCount: 10,
```

Comment:
Should we bump this? I'm thinking a scenario where a supernode goes down and it needs to re-auth all its peers simultaneously.

Reply:
Yeah possibly